_includes/themes/restart/modal.html

```diff
@@ -0,0 +1,35 @@
+<div class="modal fade reportthisjob-modal" id="reportthisjob-modal" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title" id="gridSystemModalLabel">Tell us a little more</h4>
+            </div>
+            <div class="modal-body">
+                <div class="tell-more">
+                    <div>
+                        <input type="radio" name="notremote" value="office">
+                    </div>
+                    <div>
+                        <span>I think this is an office/onsite job </span>
+                        <small>Ex: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."</small>
+                    </div>
+                </div>
+                <div class="tell-more">
+                    <div>
+                        <input type="radio" name="notremote" value="partially">
+                    </div>
+                    <div>
+                        <span>I think this job is partially remote </span>
+                        <small>Ex: a requirement to "work from Head Office for a minimum of 2 days per month"</small>
+                    </div>
+                </div>
+
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+                <button type="button" class="btn btn-primary">Submit</button>
```

Comment:
Change buttons according to trello ticket

Reply:
ok